webapp/crashstats/topcrashers/tests/test_views.py

```diff
@@ -170,38 +171,43 @@ def build_crash_data(**params):
                     product="Firefox",
                     version="3.0",
                     startup_crash=False,
+                    uptime=500,
                 )
             )
 
         for item in crash_data:
-            es_helper.index_crash(processed_crash=item, refresh=False)
-        es_helper.refresh()
+            preferred_es_helper.index_crash(processed_crash=item, refresh=False)
+        preferred_es_helper.refresh()
 
         startup_crash_msg = 'title="Startup Crash"'
         potential_startup_crash_msg = 'title="Potential Startup Crash"'
+        potential_startup_window_crash_msg = 'title="Potential Startup Crash, more than '
```

Comment:
I modified this test to also check `is_startup_window_crash` to cover the failure case you manually observed, and confirm that my solution for identifying boolean aggregation terms is working as expected.

Reply:
Nicely done--thank you!